depmapomics/qc/config.py

```diff
@@ -49,12 +49,18 @@ def taiga_latest_path(dataset_name):
         "dmc": taiga_latest_path("dmc-22q2-5e51"),
         "public": taiga_latest_path("public-22q2-de04"),
     },
+    "22Q2dryrun": {
+        "internal": taiga_latest_path("internal-22q2dryrun-5ffb"),
+        "ibm": taiga_latest_path("ibm-22q2dryrun-ddbe"),
+        "dmc": taiga_latest_path("dmc-22q2dryrun-0f8d"),
+        "public": taiga_latest_path("public-22q2dryrun-e259"),
+    },
 }  # release ids on taiga
 
 PORTALS = ["ibm", "dmc", "public", "internal"]  # used for 'bookkeeping' markers
-PORTAL = "dmc"  # used for 'not bookkeeping' markers
-PREV_QUARTER = "22Q1"
-NEW_QUARTER = "22Q2"
+PORTAL = "internal"  # used for 'not bookkeeping' markers
+PREV_QUARTER = "22Q2"
+NEW_QUARTER = "22Q2dryrun"
```

Comment:
all config val that would need to be changed at every release shouldn't be configs they should be just input to functions in the notebook..

Reply:
QCs are currently implemented as pytests so they don't really involve any notebooks..